app/src/androidTest/java/org/simple/clinic/patient/PatientRepositoryAndroidTest.kt

```diff
@@ -1156,6 +1156,7 @@ class PatientRepositoryAndroidTest {
   }
```

Comment:
As discussed, please inject clock to all the places where you are instantiating time or date.
Feel free to do this in your next PR.